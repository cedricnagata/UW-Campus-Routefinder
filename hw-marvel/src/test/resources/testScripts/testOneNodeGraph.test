# test parseData and findPath on one node graph

# build graph
LoadGraph oneNodeGraph oneNodeGraph.csv

# check graph is built
ListNodes oneNodeGraph
ListChildren oneNodeGraph oneNode

# check findPath
FindPath oneNodeGraph oneNode oneNode
FindPath oneNodeGraph oneNode twoNode