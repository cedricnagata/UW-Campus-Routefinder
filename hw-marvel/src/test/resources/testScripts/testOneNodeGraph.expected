# test parseData and findPath on one node graph

# build graph
loaded graph oneNodeGraph

# check graph is built
oneNodeGraph contains: oneNode
the children of oneNode in oneNodeGraph are:

# check findPath
path from oneNode to oneNode:
unknown: twoNode